src/components/core/DpUpload/DpUpload.vue

```diff
@@ -95,6 +95,60 @@ export default {
       return b ? b.pop() : ''
     },
 
+    /**
+     * Fires immediately before a file is added to the Uppy store.
+     *
+     * The hook is used here to clean file names from characters that would
+     * break the tus endpoint. This could as well be implemented within the backend.
+     *
+     * @param currentFile
+     * @param files
+     * @return {(*&{meta: (*&{name: *}), name: *})|*}
+     * @see https://github.com/transloadit/uppy/blob/main/packages/@uppy/core/src/Uppy.js#L503
+     */
+    handleOnBeforeFileAdded (currentFile, files) {
+      let fileName = currentFile.name
+
+      /*
+       * This is not an exhaustive list - the characters have been determined
+       * simply by trying out which break or do not break the tus endpoint.
+       */
+      const reservedCharacters = [
+        '?',
+        '#',
+        '&',
+        '\'',
+        '"',
+        '/',
+        ':',
+        '\\'
+      ]
+
+      /*
+       * For reasons i could not figure out, the for loop was needed here
+       * to reliably access the item that is looped over. When using forEach,
+       * strangely it always held the value of currentFile.
+       */
```

Comment:
since its totally fine to use a for-loop, I suggest to drop the comment.

Reply:
but in case somebody wants to optimize it, they'll be warned, if we leave the comment there.

Reply:
ok, removed in 5c6ee110cf4c7be365761f85dae3744c624ede27